examples/flood/Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM arm64v8/ubuntu
+# Use same Ubuntu version as `commonware-deployer`
+FROM arm64v8/ubuntu:24.04
```

Comment:
On `main`, we are getting the error:
```
/home/ubuntu/binary: /lib/aarch64-linux-gnu/libc.so.6: version `GLIBC_2.38' not found (required by /home/ubuntu/binary)
```

This is because we are compiling code for a different LTS version than we're deploying.